aws-cs-eks/.gitignore

```diff
@@ -0,0 +1,353 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
This is a lengthy `.gitignore`. I'm guessing it is necessary, but at 353 LoC, it's longer than the EKS cluster and app program at 277 LoC 🤔

Reply:
This is just copied from other CSharp examples - I agree it might be overkill :/